### ISSUE TITLE
[Bug]: "alt- Image Elements and aria-label- Form Section" missing in Some Files 

### ISSUE THREAD

**Issue by NarendraKoya999** [avinashkranjan/Amazing-Python-Scripts#2413]
### Describe the bug

- **In `index.html, script.html, page_contactis.html Files, alt attribute` is missed**
- **Also, In `page_contactus.html `File, `aria-labels are missing in Form Section`**

### Steps to reproduce the behavior

1. **Go To `"gh-pages"` Branch Data.**
2. **Observe `index.html, script.html and page_contactus.html Files. alt attribute is not properly used to Image Elements`**
3. **Also, Observe In `page_contactus.html File`, `aria-labels missing in Form Section`**

### Expected behavior

- **`Placed alt and aria-label attributes in Respective Files`**

### Device?

Desktop

### Which OS are affected?

Windows

### Enter the version of your web browser

**Version 115.0.5790.110 (Official Build)** 

### Additional context

- **`alt attribute` is much need for Image Elements. Because of In some Cases, Browsers does not read Images Properly**
- **`aria-label attribute is much need for SEO Optimization ` This helps `screen readers identify what each form field is for, improving the user experience for visually impaired users`.**

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

**Comment by NarendraKoya999** [comment]
Hi @avinashkranjan . Please review on this Issue

**Comment by NarendraKoya999** [comment]
Hi @avinashkranjan . Please Review on this Issue.
